projects/observability/src/shared/components/topology/renderers/tooltip/topology-tooltip-popover.ts

```diff
@@ -86,9 +85,10 @@ export class TopologyTooltipPopover implements TopologyTooltip {
     }
 
     popover.updatePositionStrategy({
-      type: PopoverPositionType.Relative,
-      locationPreferences: [PopoverRelativePositionLocation.InsideTopLeft],
-      origin: this.container
+      type: PopoverPositionType.FollowMouse,
```

Comment:
So this would follow the mouse, but how do we then close the popover? You had mentioned that relative anchored on the node would look better and I tend to agree - should we go with that option?

Reply:
I think this can work as an improvement, I would like to know what @anandtiwary thinks, but I think it solves in almost all cases the problem we had.

Reply:
Looks fine. Could you also please share screenshot of topology when you click on a node and when you hover on a node?

Reply:
I adjusted some changes that @aaron-steinfeld suggested to me and leave the evidence in the comment (images of the cases)